packages/orbit-components/src/Heading/__tests__/index.test.tsx

```diff
@@ -25,64 +18,167 @@ describe("Heading", () => {
     expect(heading).toHaveAttribute("id", "id");
   });
 
-  it("should ignore unsupported props", () => {
+  it("should have expected styles from type", () => {
     render(
-      // className has to be undefined to reproduce the error
-      // @ts-expect-error className is not defined
-      <Heading as="h1" type="display" className={undefined}>
-        My lovely heading
-      </Heading>,
+      <>
+        <Heading dataTest="Title1" type={TYPE_OPTIONS.TITLE1}>
+          Title 1
+        </Heading>
+        <Heading dataTest="Title2" type={TYPE_OPTIONS.TITLE2}>
+          Title 2
+        </Heading>
+        <Heading dataTest="Title3" type={TYPE_OPTIONS.TITLE3}>
+          Title 3
+        </Heading>
+        <Heading dataTest="Title4" type={TYPE_OPTIONS.TITLE4}>
+          Title 4
+        </Heading>
+        <Heading dataTest="Title5" type={TYPE_OPTIONS.TITLE5}>
+          Title 5
+        </Heading>
+        <Heading dataTest="Title6" type={TYPE_OPTIONS.TITLE6}>
+          Title 6
+        </Heading>
+        <Heading dataTest="Display" type={TYPE_OPTIONS.DISPLAY}>
+          Title Display
+        </Heading>
+        <Heading dataTest="DisplaySubtitle" type={TYPE_OPTIONS.DISPLAYSUBTITLE}>
+          Title DisplaySubtitle
+        </Heading>
+      </>,
     );
-    expect(screen.getByRole("heading", { name: "My lovely heading" }));
+
+    const elements = screen.getAllByText(/Title/);
+    elements.forEach(element => {
+      const type = element.getAttribute("data-test");
+      expect(element).toHaveStyle({
+        fontSize: theme.orbit[`fontSizeHeading${type}`],
+        fontWeight: theme.orbit[`fontWeightHeading${type}`],
+        lineHeight: theme.orbit[`lineHeightHeading${type}`],
+      });
+    });
+  });
+
+  it("should align text", () => {
+    render(
+      <>
+        <Heading dataTest="start" align="start">
+          start
+        </Heading>
+        <Heading dataTest="center" align="center">
+          center
+        </Heading>
+        <Heading dataTest="end" align="end">
+          end
+        </Heading>
+        <Heading dataTest="justify" align="justify">
+          justify
+        </Heading>
+      </>,
+    );
+
+    expect(screen.getByTestId("start")).toHaveStyle({
+      textAlign: "start",
+    });
+    expect(screen.getByTestId("center")).toHaveStyle({
+      textAlign: "center",
+    });
+    expect(screen.getByTestId("end")).toHaveStyle({
+      textAlign: "end",
+    });
+    expect(screen.getByTestId("justify")).toHaveStyle({
+      textAlign: "justify",
+    });
+  });
+
+  it("should have space after", () => {
+    render(
```

Comment:
That does not seem to work. The SPACINGS_AFTER enum matches to strings only, not to values for the margin.
They come from tokens whose names do not match directly the spaceAfter string value:

```
spaceAfter="none" => marginBottom: 0
spaceAfter="smallest" => marginBottom: theme.orbit.spaceXXSmall (2px)
...
```

So I don't find a way of matching it. It can probably be abstracted somewhere else to match the string to the tokens… but I think that does not simplify

Reply:
![Screenshot 2023-07-21 at 11 45 54](https://github.com/kiwicom/orbit/assets/14054752/e47e1f64-f2f0-401b-b90e-53c953e78acd)

just create object like this then :)

Reply:
![Screenshot 2023-07-21 at 11 47 57](https://github.com/kiwicom/orbit/assets/14054752/d8979547-3051-4655-bd30-abd509c0e1c6)

only adjust the type

Reply:
Ok, I have added that object on the test-utils file, since we can expect it to be used on other components as well 👍